### DISCUSSION TITLE
can I use this to sign git commits?

### DISCUSSION THREAD

**Discussion by kishaningithub** [FiloSottile/age#372] [Q&A] [answered]
People generally use GPG to sign git commits. Can this be used to sign git commits?

If so, can you provide some examples?

Thanks in advance

**Comment by mohammed90** [comment, marked as answer]
age is targeting the encryption use case, not signing. For signing, you can look at signify or minisign.

**Reply by kishaningithub** [reply]
If possible can you put links for those two tools? Thanks in advance

**Reply by covert-encryption** [reply]
> If possible can you put links for those two tools? Thanks in advance

https://jedisct1.github.io/minisign/

Just use Minisign. The two used to be the same but Minisign later fixed some issues in the original design, and AFAIK Signify has not been updated with these fixes (like doing blake2b hashing of data before signing, where the legacy implementation signed data directly).
